src/raw.rs

```diff
@@ -592,6 +592,52 @@ pub fn set_bitmap(data: &[u8]) -> SysResult<()> {
     Ok(())
 }
 
+
+///Set files to clipboard.
+pub fn set_file_list(paths: &str) -> SysResult<()> {
+    let final_str = paths;
+    let files_bytes: Vec<u16> = final_str.encode_utf16().collect();
```

Comment:
use `MultiByteToWideChar` to convert to wide string format.

you can avoid creating intermediate buffer this way

To determine size of output you can use it with empty output buffer like here:
https://github.com/DoumanAsh/clipboard-win/blob/master/src/raw.rs#L336-L338